esmock 1.9.1 and eslint: Unable to resolve path to module 'esmock'

With 1.9.1 I get:

```
> eslint .


/test/my.test.js
  17:20  error  Unable to resolve path to module 'esmock'  import/no-unresolved
```

the problem are the `exports` added in: https://github.com/iambumblehead/esmock/commit/96d2754ec0974a29d8323e3122515c4d0bbd77ea

also see:
https://nodejs.org/docs/latest-v16.x/api/packages.html#package-entry-points


when I add the 

```
  "main": "./src/esmockLoader.js",
```

again, it works.

--- a/package.json
+++ b/package.json
@@ -6,6 +6,7 @@
   "readmeFilename": "README.md",
   "description": "provides native ESM import mocking for unit tests",
   "author": "chris <chris@bumblehead.com>",
+  "main": "./src/esmockLoader.js",
   "exports": {
     "types": "./src/esmock.d.ts",
     "import": "./src/esmockLoader.js"

